platform-core-module/src/main/java/com/softicar/platform/core/module/user/UserPreferences.java

```diff
@@ -0,0 +1,27 @@
+package com.softicar.platform.core.module.user;
+
+public class UserPreferences {
+
+	public boolean automaticallyCollapseFolders = false;
+	public boolean recursivelyCollapseFolders = false;
```

Comment:
we should either make them private or remove the setters and getters, IMO

Reply:
ok